### PR TITLE
Profile stack added

### PR DESCRIPTION
**Pull request by lessly-zuniga**
The following screens are added:
 *initial screen, with redirect buttons to forms to edit profile, 
 *the forms where the information is edited, 
 *2 buttonSheets, one to change the profile photo, and the other to show a QR
 *and logout alert
 
Simulator Screen Recording - iPhone 15 Pro Max - 2023-10-05 at 10 54 37

Simulator Screen Recording - iPhone 15 Pro Max - 2023-10-05 at 10 55 32

### CODE REVIEW

File: src/components/molecules/AppVersion.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { TextStyle } from 'react-native';
+import { useTranslation } from 'react-i18next';
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (EdgardoV) [inline src/components/molecules/AppVersion.tsx:4]: @lessly-zuniga Why this `eslint-disable`?

File: src/components/molecules/ProfilePhoto.tsx [hunk 2]
```diff
@@ -2,56 +2,105 @@ import React from 'react';
...
-  fadeIn, size, withName, cantEdit,
+  fadeIn, size, withName, canEdit, withId, onPress,
 }: Props) => {
   // const { photo, name, lastName } = useSelector((state: RootState) => state.profile);
 
   // eslint-disable-next-line max-len
   const photo = 'https://images.pexels.com/photos/220453/pexels-photo-220453.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2';
-  const name = 'Andrés';
-  const lastName = 'L99';
+  // const photo = '';
```
Comment (EdgardoV) [inline src/components/molecules/ProfilePhoto.tsx:25]: @lessly-zuniga Remove this comment

File: src/components/molecules/ProfilePhoto.tsx [hunk 3]
```diff
@@ -2,56 +2,105 @@ import React from 'react';
...
+  canEdit?: boolean;
+  withId?: boolean;
+  onPress?: () => void;
 }
 
 const ProfilePhoto: React.FC<Props> = ({
-  fadeIn, size, withName, cantEdit,
+  fadeIn, size, withName, canEdit, withId, onPress,
 }: Props) => {
   // const { photo, name, lastName } = useSelector((state: RootState) => state.profile);
```
Comment (EdgardoV) [inline src/components/molecules/ProfilePhoto.tsx:21]: @lessly-zuniga Remove this comment or adding TODO

File: src/components/molecules/ProfilePhoto.tsx [hunk 4]
```diff
@@ -2,56 +2,105 @@ import React from 'react';
...
         }}
       >
-        <EditIcon color={Theme.Colors.DarkSoul} height={16} width={16} />
+        <Touchable onPress={onPress}>
+          <EditIcon color={Theme.Colors.DarkSoul} height={16} width={16} />
+        </Touchable>
       </Container>
       )}
+      {withName && (
+      <CustomText
```
Comment (EdgardoV) [inline src/components/molecules/ProfilePhoto.tsx:81]: @lessly-zuniga Try to use CustomText as Text
```suggestion
      <Text
```

File: src/components/organisms/ButtonFieldEdit.tsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { CustomText } from 'components/atoms/CustomText';
+import Theme from 'theme';
+import { EditIcon } from 'assets/svg';
+import { Touchable } from 'components/atoms';
+import { Container } from '../atoms/Container';
```
Comment (EdgardoV) [inline src/components/organisms/ButtonFieldEdit.tsx:6]: @lessly-zuniga You're adding an import for Touchable from `'components/atoms'` and you have another one just for Container. 
```suggestion
import { Container } from '../atoms/Container';
import { Touchable } from '../atoms/Touchable';
```

File: src/components/organisms/EditProfileDataForm.tsx [hunk 6]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { CustomText } from 'components/atoms/CustomText';
+import Theme from 'theme';
+import { Input } from 'components/atoms';
+import { Button } from 'components/molecules';
+import { useTranslation } from 'react-i18next';
+import { Container } from '../atoms/Container';
```
Comment (EdgardoV) [inline src/components/organisms/EditProfileDataForm.tsx:7]: @lessly-zuniga Same here, try avoid to use different types of imports

File: src/components/organisms/ProfileEditForm.tsx [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+    if (field.type === 'phone' || field.type === 'email') {
+      navigate('ConfirmationCode', { type: field.type, value: field.value });
+    }
+    setShowFormEdit('');
+  };
+
+  return (
+    <Container style={{ padding: Theme.Sizes.Padding }}>
+      {fields.map((field, index) => (
+        <React.Fragment key={index}>
```
Comment (EdgardoV) [inline src/components/organisms/ProfileEditForm.tsx:61]: @lessly-zuniga Why you didn't use just a <></>?

File: src/components/organisms/index.ts [hunk 8]
```diff
@@ -1,2 +1,5 @@
 export * from './Tab';
 export * from './Alert';
+export * from './ProfileEditForm';
+export * from './ButtonFieldEdit';
+export * from './EditProfileDataForm';
```
Comment (EdgardoV) [inline src/components/organisms/index.ts:5]: @lessly-zuniga These imports are not sorted alphabetically.

File: src/screens/app/profile/ConfirmationCode/ConfirmationCodeController.tsx [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+  value: string;
+}
+
+const ConfirmationCodeController: React.FC = () => {
+  const { goBack } = useNavigation();
+  const route = useRoute<RouteProp<Record<string, RouteParams>>>();
+  const { type, value } = route.params;
+  return (
+    <SafeArea>
+      <ConfirmationCodeScreen onSubmit={() => { goBack(); }} type={type} value={value} />
```
Comment (EdgardoV) [inline src/screens/app/profile/ConfirmationCode/ConfirmationCodeController.tsx:17]: @lessly-zuniga You could pass the reference of function
```suggestion
      <ConfirmationCodeScreen onSubmit={goBack} type={type} value={value} />
```

File: src/components/molecules/AppVersion.tsx (hunk 1 above)
Comment (EmmanuelSalazarID) [inline src/components/molecules/AppVersion.tsx:4]: Remove this eslint-disable

File: src/screens/app/profile/Profile/ProfileScreen.tsx [hunk 10]
```diff
@@ -0,0 +1,143 @@
...
+        width={40}
+        height={40}
+        style={{ backgroundColor: Theme.Colors.PlaceboBlue, borderRadius: 14, justifyContent: 'center' }}
+      >
+        {icon}
+      </Container>
+    )}
+    alignContent="flex-start"
+  />
+);
```
Comment (EdgardoV) [inline src/screens/app/profile/Profile/ProfileScreen.tsx:55]: @lessly-zuniga If this component only will be used here you leave there or try to move to global components or create a components folder inside Profile folder.

**Review by EdgardoV — CHANGES_REQUESTED** [review]

**Pull request closed without merging**